subprojects/core/src/test/groovy/org/gradle/initialization/MixInLegacyTypesClassLoaderTest.groovy

```diff
@@ -78,13 +78,15 @@ class MixInLegacyTypesClassLoaderTest extends Specification {
             class JavaPluginConvention {
                 public static final String SOME_CONST = "Value";
                 public static final String SOME_CONST_WITH_GETTER = "Other Value";
+                public static final String SOME_CONST_WITH_RHS_EXPRESSION = doSomething("Derived Value");
```

Comment:
Should we check this in the case with a getter as well?

Reply:
I will add one